ui/app/components/clients/attribution.js

```diff
@@ -4,48 +4,61 @@ import { tracked } from '@glimmer/tracking';
 /**
  * @module Attribution
  * Attribution components display the top 10 total client counts for namespaces or auth methods (mounts) during a billing period.
- * If view is filtered for a single month, two graphs display and show a breakdown of new and total client counts by namespace or auth method, respectively 
+ * A horizontal bar chart shows on the right, with the top namespace/auth method and respective client totals on the left.
  *
  * @example
  * ```js
- * <Clients::Attribution
- *    @newClientsData={{this.newClientData}}
- *    @totalClientsData={{this.totalClientData}}
+ *  <Clients::Attribution
  *    @chartLegend={{this.chartLegend}}
+ *    @topTenNamespaces={{this.topTenNamespaces}}
+ *    @runningTotals={{this.runningTotals}}
+ *    @selectedNamespace={{this.selectedNamespace}}
+ *    @startTimeDisplay={{this.startTimeDisplay}}
+ *    @endTimeDisplay={{this.endTimeDisplay}}
  *    @isDateRange={{this.isDateRange}}
- *    @isAllNamespaces={{this.isAllNamespaces}}
-    >
-      <button type="button"> Export attribution data </button>
-    </Clients::Attribution>
+ *  />
  * ```
-
- * @param {array} newClientsData - (passed to child chart) must be an array of flattened objects
- * @param {array} totalClientsData - (passed to child chart) must be an array of flattened objects
  * @param {array} chartLegend - (passed to child) array of objects with key names 'key' and 'label' so data can be stacked
- * @param {boolean} isDateRange - discerns if dataset from API is a date range or single month
- * @param {boolean} isAllNamespaces - relays if filtered by all namespaces, or by single namespace
- * @param {string} activityDateRange - for modal and csv download to display correct date range
+ * @param {array} topTenNamespaces - (passed to child chart) array of top 10 namespace objects
+ * @param {object} runningTotals - object with total client counts for chart tooltip text
+ * @param {string} selectedNamespace - namespace selected from filter bar
+ * @param {string} startTimeDisplay - start date for CSV modal
+ * @param {string} endTimeDisplay - end date for CSV modal
+ * @param {boolean} isDateRange - getter calculated in parent to relay if dataset is a date range or single month
  */
 
 export default class Attribution extends Component {
   @tracked showCSVDownloadModal = false;
 
-  // TODO CMB should this be a getter? It will be updated from parent component's filter bar
   get isDateRange() {
     return this.args.isDateRange;
   }
 
-  get isAllNamespaces() {
-    return this.args.isAllNamespaces;
+  get isSingleNamespace() {
+    // if a namespace is selected, then we're viewing top 10 auth methods (mounts)
+    return !!this.args.selectedNamespace;
+  }
+
+  get totalClientsData() {
```

Comment:
name is `totalClientsData` (opposed to something like `barChartData`) because in 1.11 there will be side by side charts displaying new clients **and** total clients. so preparing to have a parallel getter `newClientsData` in the future with top 10 new clients